### DISCUSSION TITLE
pixel values

### DISCUSSION THREAD

**Discussion by krhodes9** [gee-community/qgis-earthengine-plugin#99] [Q&A]
Do you have any suggestions for viewing/extracting the feature values from the the images brought into QGIS using the plug-in? I am working over a large study area and all the pixels only display a value of no-data.

**Comment by gena** [comment]
Querying a value manually should work out of the box, but you will have to use EE API to query many values, e.g. reduceRegion + ee.Reducer.toList or getRegion or export. 

Or do you mean FeatureCollections?

![image](https://user-images.githubusercontent.com/169821/124235166-367eae00-db15-11eb-9ded-4461623169b2.png)

**Reply by krhodes9** [reply]
I am trying to create a monthly composite image within a defined region of interest. The image is created, but there are no values showing for any of the pixels.

![image](https://user-images.githubusercontent.com/61336730/124298666-23260f80-db22-11eb-83ae-2009edd77142.png)

**Reply by gena** [reply]
Sorry for the very long delay, did the issue resolve? Otherwise, can you share your python script to debug?

**Reply by wkordo** [reply]
I am currently having the same issue. I am using the WWF Hydrosheds image, and it creates the image as desired, however when I inspect a pixel, it does not return a value for b1 to indicate direction. I have attached an image and my python script. 

![Drainage direction troubleshoot](https://user-images.githubusercontent.com/109970853/180799563-6ddbbc14-a00c-42d5-bb8f-ffc5dcd995ae.png)



import ee
from ee_plugin import Map

image = ee.Image('WWF/HydroSHEDS/03DIR');

vizParams = {
  'bands': ['b1'],
  'min': 1,
  'max': 128,
  'palette': ['000000', '023858', '006837', '1a9850', '66bd63', 'a6d96a', 'd9ef8b',
    'ffffbf', 'fee08b', 'fdae61', 'f46d43', 'd73027']
 
};

Map.setCenter(-88.48236,47.16566, 15); 
Map.addLayer(image, vizParams, 'Drainage Direction');

**Comment by gena** [comment]
That's strange, I've just tested (Win11, QGIS 3.18, EE plugin 0.0.5) and it seems to work just fine 
![image](https://user-images.githubusercontent.com/169821/180861892-b06f24ec-b456-4c8f-aef7-563769fb15c1.png)

Can you double-check with the same experiment, after making sure you have the latest version of the plugin and QGIS installed (start QGIS, create a new map, run the script, select the newly added layer, click inspect with the info tool)?

**Reply by wkordo** [reply]
Running the same script in QGIS 3.22.3 and EE plugin 0.0.5 I now cannot gather any information from identify features tool. When clicking any point on the layer it returns  "no features found at this location" which can be seen in the bottom left.

![image](https://user-images.githubusercontent.com/109970853/181012581-c7731fe8-ad50-45be-bc20-1f6f56ac830a.png)

**Comment by gena** [comment]
Can you try changing the map projection to EPSG:3857? I think this is probably the reason.

**Reply by wkordo** [reply]
This works with QGIS version 3.16, but not 3.22. Thank you, that is enough for me to work with.

**Reply by gena** [reply]
Happy to hear that the workaround works. It's a bug and it will be fixed. Thanks for reporting!

**Comment by spatialthoughts** [comment]
For QGIS 3.22 the workaround also doesn't work. You get 'no features found at this location' even after setting the CRS to 3857.

**Comment by gena** [comment]
Oh! Thanks for letting me know, it looks like it's time for a new bugfix
release.

**Comment by gena** [comment]
Should be fixed in v0.0.6
